Client.xcodeproj/project.pbxproj

```diff
@@ -5880,7 +5898,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = NO;
-				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Extensions/Entitlements/FennecEnterprise.entitlements";
+				CODE_SIGN_ENTITLEMENTS = Extensions/Entitlements/FennecEnterprise.entitlements;
```

Comment:
(inherit)  is missing here. Is this on purpose?

Reply:
mm not on purpose, let me check if I missed that one...

Reply:
Sorry about that, I missed this one and another too, I fixed that. Although for some reason builds were not affected and were built and run the tests successfully on both CI tools....